RealmSwift-swift2.0/List.swift

```diff
@@ -407,9 +407,36 @@ public final class List<T: Object>: ListBase {
 
     The block will be asynchronously called with the initial list, and then
     called again after each write transaction which changes the list or any of
-    the items in the list. You must retain the returned token for as long as
-    you want the results to continue to be sent to the block. To stop receiving
-    updates, call stop() on the token.
+    the items in the list.
+     
+    The block is called on the same thread as it was added on, and can only
+    be added on threads which are currently within a run loop. Unless you are
+    specifically creating and running a run loop on a background thread, this
+    normally will only be the main thread.
+     
+    Notifications can't be delivered as long as the runloop is blocked by
+    other activity. When notifications can't be delivered instantly, multiple
+    notifications may be coalesced. That can include the notification about the
+    initial list.
```

Comment:
Notifications cannot be delivered while the runloop is being blocked by other activity. When notifications aren't able to be delivered instantly, they may be coalesced. That can even apply to the initial notification that is sent after the block is registered.